client/src/pages/PlayerScreen.tsx

```diff
@@ -1,24 +1,50 @@
+import { useEffect, useState } from "react";
 import DisplayLogo from "../components/DisplayLogo";
 import WaitingToStart from "../components/WaitingToStart";
-//import ChoosePlayerMove from "../components/ChoosePlayerMove";
-//import CountDownTimer from "../components/CountDownTimer";
+import { socket } from "../App";
+import ChoosePlayerMove from "../components/ChoosePlayerMove";
+import CountDownTimer from "../components/CountDownTimer";
+import WinnerPlayer from "../components/WinnerPlayer";
 
 const PlayerScreen = () => {
+  const [shouldRenderComponents, setShouldRenderComponents] = useState(false);
```

Comment:
probably should be a bit more descriptive, like what components are being rendered? Also could be shortened, to maybe just renderComponents or something, but that doesnt really matter.